### DISCUSSION TITLE
Customizing location with silent installation

### DISCUSSION THREAD

**Discussion by guyas** [SafeExamBrowser/seb-win-refactoring#803] [Q&A]
Hi,
In our project the browser has its own name, logo etc (defined in the NSIS installer setting file).  That is, we define custom `INSTDIR` and `SEB_PATH` variables so all files should be installed under `C:\Program Files\ABC`. After a few more commands we run
`	ExecWait '"$INSTDIR\installation\SEB_3.6.0.633_SetupBundle.exe" /quiet /norestart'
`
While all other stuff does go to `C:\Program Files\ABC`, the SEB files (with subdirectories `Application`, `Configuration`, `Reset`, `Service`) go under `C:\Program Files\SafeExamBrowser`, which I can only assume is the default installation path, especially as this is a quiet installation.

Is there a way to provide a flag that customizes the installation location?

**Comment by dbuechel** [comment]
At the top of my head I do not know of any. But you may find a parameter on the Windows Installer XML (WiX Toolset) website resp. documentation: https://wixtoolset.org/.
